demo/index.js

```diff
@@ -1,77 +1,306 @@
+
+            task.classList.add("drag-over");
+        });
+        task.addEventListener("drop", (event) => {
+            event.preventDefault();
+            event.stopPropagation();
+            event.stopImmediatePropagation();
+
+            task.classList.remove("drag-over");
+
+            if (currentlyDraggedTask !== task) {
+                swapDom(currentlyDraggedTask, task);
+            }
+        });
+        task.addEventListener("dragleave", () => {
+            task.classList.remove("drag-over");
+        });
+        task.addEventListener("dragover", (event) => event.preventDefault());
+        task.addEventListener("dragstart", () => {
+            currentlyDraggedTask = task;
+        });
+        task.addEventListener("dragend", () => {
+            currentlyDraggedTask = null;
+        });
+    }
 }
+
+enableDragForTasks();
+
+function enableDragForKanbanBoard() {
+    let currentlyDraggedTask = null;
+
+    const blocks = document.querySelectorAll(`#kanban-board .block`);
+    for (let i = 0; i < blocks.length; i++) {
+        const block = blocks[i];
+        const tasks = block.querySelectorAll(".task");
+
+        for (let j = 0; j < tasks.length; j++) {
+            const task = tasks[j];
+
+            task.addEventListener("dragstart", () => {
+                currentlyDraggedTask = task;
+            });
+
+            task.addEventListener("dragend", () => {
+                currentlyDraggedTask = null;
+            });
+
+            task.addEventListener("dragover", (event) => {
+                event.preventDefault();
+                task.classList.add("drag-over");
+            });
+
+            task.addEventListener("dragleave", () => {
+                task.classList.remove("drag-over");
+            });
+
+            task.addEventListener("drop", (event) => {
+                event.preventDefault();
+                event.stopPropagation();
+
+                task.classList.remove("drag-over");
+                blocks.forEach((block) => block.classList.remove("drag-over"));
+
+                if (currentlyDraggedTask && currentlyDraggedTask !== task) {
+                    swapDom(currentlyDraggedTask, task);
+                }
+            });
+        }
+
+        block.addEventListener("dragover", (event) => {
+            event.preventDefault();
+            block.classList.add("drag-over");
+        });
+
+        block.addEventListener("dragleave", () => {
+            block.classList.remove("drag-over");
+        });
+
+        block.addEventListener("drop", (event) => {
+            event.preventDefault();
+            event.stopPropagation();
+
+            block.classList.remove("drag-over");
+
+            if (currentlyDraggedTask) {
+                const ul = block.querySelector("ul");
+                const tasks = block.querySelectorAll(".task");
+        
+                // Get mouse position
+                const mouseY = event.clientY;
```

Comment:
No need to rename it:
```suggestion
                const { clientY } = event;
```